src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -44,7 +44,8 @@ private static GitPullRequest CreatePullRequest(string sourceBranch, string targ
                 Title = $"{prefix}{Options.InsertionName} '{Options.BranchName}/{buildToInsert}' Insertion into {Options.VisualStudioBranchName}",
                 Description = description,
                 SourceRefName = sourceBranch,
-                TargetRefName = targetBranch
+                TargetRefName = targetBranch,
+                IsDraft = true,
```

Comment:
What's the effect in practice of making the insertion PR a draft? Reduces notification noise?

Reply:
It does. Need to remember who was asking for this. Maybe @jasonmalinowski remembers

Reply:
A thing to think about is that when an insertion pr is marked as a draft, none of the policies run. Manually queuing a policy still won't run it.

Reply:
So it seemed like this would be useful for DummyPRs, but you also can't approve a draft PR which makes it less than ideal

Reply:
I think we did want draft support so anybody making insertions for their own uses (to run RPS) could be marked as draft so nobody gets excited pressing merge buttons. "Normal" insertions would not use draft.

Reply:
I could not get any policies to run while in Draft state. Cloudbuild, DDRIT, Symbol check, etc....

Reply:
I came back around to this and draft PRs can now launch policies to run. They do not start automatically so I will look into starting them with RIT.

Reply:
Updated so that specifying /QueueValidationBuild=true will start all the checks for draft PRs now.